packages/components/input/Field.ts

```diff
@@ -166,25 +193,29 @@ export abstract class Field<T> extends Input<T> {
 			:host([dense]) {
 				--mo-field-height: 32px;
 				--mo-field-label-scale-value-on-focus: 1;
+				--mo-field-label-top-on-focus: 16px;
 			}
 
-			:host([disabled]) div[part=container], :host([disabled]) slot[name=trailingInternal] {
+			:host([disabled]) {
```

Comment:
Das kann eine gute Idee sein oder doch Schiff gehen.

Früher hatte `div[part=container]` `pointer-events: none` und nicht das ganze Feld, damit man die Freiheit hat, nicht-deaktivierte Elemente in das Field slottet und mit denen interagiert, obwohl der Inhalt des Fields deaktiviert ist. 

Bestätige daher bitte erst, dass kein Field in eBusiness dieses benutzt.

Reply:
Ich dachte, wenn ein Feld disabled ist, sollten keine Aktionen möglich sein, auch nicht an den Buttons, die am Feld sind, da der Inhalt nicht verändert werden soll. Oder verstehe ich das falsch? Wenn z.B. ein Datum fest ist und nicht verändert werden darf, aber das deaktivierte Feld würde das zulassen, würde das dann nicht die Daten verändern?

Reply:
Es geht nicht um die neue Funktion.
Es geht darum, dass das früher bewusst so gemacht wurde, dass die Custom-Fields die Freiheit haben für sich zu entscheiden, ob diese `disabled` vom Feld übernehmen oder doch nicht. Bei dieser Änderung sollst du also in eBusiness kurz gucken, ob so einen Fall existiert, und ob irgendeine Komponente mit dieser Option arbeitet, oder nicht.

Reply:
hm, irgendwie verstehe ich das nicht ganz. Das muss du mir vielleicht zeigen.